es-core/src/Log.h

```diff
@@ -5,8 +5,8 @@
 #include <sstream>
 
 #define LOG(level) \
-if(level > Log::getReportingLevel()) ; \
-else Log().get(level)
+if(level <= Log::getReportingLevel()) \
```

Comment:
Changing to this logic reduces the number of dangling else warnings to a manageable level.